serveradmin/serverdb/migrations/0013_change_index.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.16 on 2022-11-03 16:12
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('serverdb', '0012_migrate_change_tables'),
+    ]
+
+    operations = [
+        # Apparently Django doesn't support creating indexes on jsonb keys yet
+        migrations.RunSQL(
+            """
+            CREATE INDEX
```

Comment:
I would rather not run it concurrently in an automated migration as it can fail silently but I think it's a good idea to run it via CREATE INDEX IF EXISTS and run it manually beforehand with CONCURRENTLY - good idea.